src/plugins/news/components/story-fields/index.jsx

```diff
@@ -1,75 +1,105 @@
 import { Card, CardBody, CardHeader } from '@triniti/admin-ui-plugin/components';
 import { Field, FieldArray } from 'redux-form';
-import slottingConfig from 'config/slottingConfig'; // eslint-disable-line import/no-unresolved
+import { formRules } from '@triniti/cms/plugins/news/constants';
 import DatePickerField from '@triniti/cms/components/date-picker-field';
+import get from 'lodash/get';
 import KeyValuesField from '@triniti/cms/components/key-values-field';
 import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';
 import PicklistPickerField from '@triniti/cms/plugins/sys/components/picklist-picker-field';
 import PropTypes from 'prop-types';
 import React from 'react';
 import Schema from '@gdbots/pbj/Schema';
+import slottingConfig from 'config/slottingConfig'; // eslint-disable-line import/no-unresolved
 import SlugEditor from '@triniti/cms/plugins/ncr/components/slug-editor';
 import TextField from '@triniti/cms/components/text-field';
 
 const StoryFields = ({
```

Comment:
@aloncohenn remember this component will also need to be updated at the vendor level because it is being overwritten